services/api/src/routes/organizations.js

```diff
@@ -0,0 +1,77 @@
+const Router = require('@koa/router');
+const { validateBody } = require('../utils/middleware/validate');
+const { authenticate, fetchUser } = require('../utils/middleware/authenticate');
+const { requirePermissions } = require('../utils/middleware/permissions');
+const { userHasAccess } = require('../utils/permissions');
+const { Organization } = require('../models');
+
+const router = new Router();
+
+router
+  .use(authenticate({ type: 'user' }))
+  .use(fetchUser)
+  .param('organizationId', async (id, ctx, next) => {
+    const organization = await Organization.findById(id);
+    ctx.state.organization = organization;
+    if (!organization) {
+      ctx.throw(404);
+    }
+    return next();
+  })
+  .post('/mine/search', authenticate({ type: 'user' }), fetchUser, async (ctx) => {
```

Comment:
Oh interesting... I never questioned why search was a POST I assumed it was to do with URL length limitations, I didn't realize it was for nested objects. So looking at it `koa-qs` gives us nested URL params right? So is there a need for any search routes to be a POST?

At any rate to answer your question because it's a `/search` route I gave it parity and am using the search utils so it DOES in fact allow filtering... I'm more of the thinking that everything should be handled by a single `/search` route but that's just me

Reply:
Yeah I think koa-qs (but we should confirm that we can do e.g. `?foo[bar]=1` which should translate to `{ foo:{ bar: 1 } }`

I think it make sense to split the use cases 
But I assume the pattern is generally the search is for privilege access and more of admin search (can be slow and not meant to be cached) 

While the /mine or what ever are more public facing with less filtering and more scoped access. 

I think the those 2 use-cases are much easier to split apart with 2 routes because it makes the endpoints much easier to understand/maintain because thats how the middleware is designed and how the query is generate for the 2 different use cases.

Reply:
I dunno I'll leave this up to @dominiek 

1. I'm fine with POST... there are URL length limits that might be an issue (but probably not) but I don't really see a hard argument either way so whatever is fine
2. Now that we can easily filter why not just do it? Makes bedrock more usable out of the box.

Reply:
Don't let this block you.
We could split it out in a different PR or push it down the road, its not like we don't have loose ends bedrock already.

Reply:
ok leaving this one for now... whether `/search` is a POST or not is out of scope for this PR and we can discuss later